### DISCUSSION TITLE
[SUGGESTION] Disable the annoying firefox built-in translator

### DISCUSSION THREAD

**Discussion by guilhermerek** [yokoffing/Betterfox#256] [Ideas]
Firefox new translator can be annoying opening sites in other idioms. Like opening github.
![image](https://github.com/yokoffing/Betterfox/assets/50538146/011d6485-dafc-40d6-927e-96e4492e0781)

I think this should work to disable (in my case it worked):
user_pref("browser.translations.automaticallyPopup", false);
user_pref("browser.translations.enable", false);

*I'm sorry for my English, it's not my primary language*

**Comment by yokoffing** [comment]
I'd rather not. People complained for years that Firefox couldn't match Chrome without a Language Translation feature, and now they finally implemented it.

It's already in Peskyfox as an option. Just `copy`+`paste`, uncomment, and change to `false`:

https://github.com/yokoffing/Betterfox/blob/6657c83ae202f990c77858d2bc852f5f9cb73683/Peskyfox.js#L130-L138

**Reply by guilhermerek** [reply]
Thank you for the answer, I didn't know about the translation feature. Thanks!
